app/lib/steps/partner_capital_section.rb

```diff
@@ -2,19 +2,15 @@ module Steps
   class PartnerCapitalSection
     PROPERTY_STEPS = %i[partner_property partner_property_entry].freeze
     VEHICLE_STEPS = %i[partner_vehicle partner_vehicle_details].freeze
-    TAIL_STEPS = %i[partner_assets].freeze
+    TAIL_STEPS = %i[partner_details partner_assets].freeze
```

Comment:
I think this is a typo - when the flag is enabled, PartnerCapitalSection isn't used, when it's not enabled, partner_details are provided by the PartnerSection.

Reply:
yes sorry hangover from when I thought details should be bewteen assets pages